### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: rpcs3/util/asm.hpp [hunk 1]
```diff
@@ -366,24 +366,24 @@ namespace utils
 	}
 
 	// Align to power of 2
-	template <typename T, typename = std::enable_if_t<std::is_integral<T>::value && std::is_unsigned<T>::value>>
```
Comment (elad335) [inline rpcs3/util/asm.hpp:369]: enum can be unsigned, bool is unsigned but they are not integral.

Comment (Nekotekina) [reply rpcs3/util/asm.hpp:369]: No, see https://en.cppreference.com/w/cpp/types/is_unsigned it even has examples with enums.
Only integral types can return true here.

**Comment by AniLeo** [comment]
test

**Comment by AniLeo** [comment]
test

**Pull request merged**